packages/manager/apps/web/client/app/configuration/configuration.controller.js

```diff
@@ -17,7 +18,7 @@ angular.module('App').controller(
       this.allGuides = get(
         this.constants,
         `urls.${this.subsidiary}.guides.all`,
-        this.constants.urls.FR.guides.all,
+        this.constants.urls[Environment.getRegion()].guides.all,
```

Comment:
Is it legit to replace `FR` by `EU`? I don't see any `EU` in `URLS` (see packages/manager/apps/web/client/app/config/constants.config.js)

Reply:
No you're right. It has been reverted and test has been updated to prevent having `TypeError`.